admin/settings.php

```diff
@@ -126,23 +126,30 @@ public function init_settings() {
 				if(!isset($field["register"]) || $field["register"]) {
 					register_setting($this->option_group, $this->prefix.$field["name"]);
 				}
-				
+
 			}
 		}
 	}
-	
+
 	/**
 	 * Add Settings submenu and page
 	 *
 	 * @since  0.10
 	 * @return void
 	 */
 	public function add_settings_menu() {
+
+		$cap = RUA_App::CAPABILITY;
+		if ( ! is_multisite() && is_super_admin() && ! current_user_can( $cap ) ) {
+			// Replicate multisite super admin behavior for single installations.
+			$cap = 'manage_options';
```

Comment:
If we go with this, I think a better solution would be to create a new function, is_level_admin, that basically just wraps the above, and then use that throughout the plugin.

But I think it can easily become a mess if using different capabilities/checks depending on the type of the site, so I think just making sure all admins get manage_restrictions on plugin activation is better. There is already code in place to add callbacks when updating the plugin as well, so existing users of the plugin will get migrated.

And then we can filter out that capability in the level editor, so users can't elevate themselves.

What do you think?